features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/Resource.js

```diff
@@ -231,7 +238,7 @@ class Resource extends React.Component{
                             </table>
                             : ''}
                     </Col>
-                </Row>
+                </Row> : null}
```

Comment:
If no else case, we can simply use `argument && value` pattern, [example](https://stackoverflow.com/questions/11554492/javascript-shorthand-if-statement-without-the-else-portion)